.github/workflows/test.yml

```diff
@@ -23,17 +23,13 @@ on:
 
 jobs:
   ci:
-    runs-on: <image-name>
+    runs-on: ubuntu-22.04
+    container:
+      image: exercism/gdscript-test-runner
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
```

Comment:
General practice for Exercism now is to pin actions to a particular commit SHA.

Reply:
Changed to the SHA used by Crystal and Python tracks, thanks :)